Makefile

```diff
@@ -1,3 +1,4 @@
+# TODO change to roary or Roary
```

Comment:
This might be kinda tricky, because SDK apps like to repeat the name throughout the repo. You could try grepping for 'slebras' in the whole repo. Would probably be a little better if it was called "Roary", but not a big deal at all